frontend/src/types.ts

```diff
@@ -1373,6 +1373,7 @@ export interface Experiment {
     end_date?: string
     created_at: string
     created_by: UserBasicType | null
+    feature_flag_variants: MultivariateFlagVariant[]
```

Comment:
For now, let's put this information into `parameters` ? Not exactly sure how the final version will look, and if it's in parameters we can iterate the FE/BE without making model changes. And we can refactor it out in the end before releasing

Reply:
sure!